### PR TITLE
list of orderstatuses

### LINKED ISSUE #329: Consider including a list of recommended order statuses
the `orderStatus` property of an `OrderStatus` message is effectively plaintext which becomes quite difficult to wrangle especially from a wallet's perspective. We should include a list of recommended order statuses in the spec and a description of said status. 

Additionally, It would be helpful to provide these recommended order statuses as enums via our SDKs

Here's a few that come to mind immediately:

| Status             | Description                                                                              |
| :----------------- | :--------------------------------------------------------------------------------------- |
| `PAYIN_PENDING`    | Order has been received and PFI is waiting to receive the quoted payin amount from Alice |
| `PAYIN_RECEIVED`   | the quoted payin amount has been received by the PFI                                     |
| `PAYOUT_INITIATED` | the PFI has initiated the quoted payout                                                  |

### PR DESCRIPTION
**Pull request by jiyoontbd**
closes #329

### CODE REVIEW

File: hosted/json-schemas/orderstatus.schema.json [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
   "type": "object",
-  "required": [
-    "orderStatus"
-  ],
   "additionalProperties": false,
   "properties": {
-    "orderStatus": {
+    "status": {
+      "type":"string",
+      "enum": [
```
Comment (angiejones) [inline hosted/json-schemas/orderstatus.schema.json:9]: missing `PAYIN_EXPIRED`

**Review by angiejones — CHANGES_REQUESTED** [review]

File: hosted/json-schemas/orderstatus.schema.json (hunk 1 above)
Comment (jiyoontbd) [reply hosted/json-schemas/orderstatus.schema.json:9]: thanks angie! https://github.com/TBD54566975/tbdex/pull/351/commits/2b3c594a5acd31db71bc4e5cda75f01bef62d6af

**Comment by michaelneale** [comment]
looks good - hard to know if missing some obvious gap, are there any external references that we can point to that show this covering things or is this more as we have discovered so far?

**Comment by jiyoontbd** [comment]
> looks good - hard to know if missing some obvious gap, are there any external references that we can point to that show this covering things or is this more as we have discovered so far?

@michaelneale this is more of what we've discovered so far from internal testing. of course super open to adding or removing from the enum list as we keep building with tbdex. :)

**Review by angiejones — APPROVED** [review]

**Review by nitro-neal — APPROVED** [review]

**Comment by michaelneale** [comment]
@jiyoontbd SGTM - I am guessing its not too dramatic to remove later

**Review by mistermoe — APPROVED** [review]
Awesome work @jiyoontbd

**Comment by KendallWeihe** [comment]
> Awesome work @jiyoontbd

Seconded!

**Pull request merged**